sympy/integrals/tests/test_manual.py

```diff
@@ -474,3 +474,17 @@ def test_issue_15471():
     f = log(x)*cos(log(x))/x**(S(3)/4)
     F = -128*x**(1/4)*sin(log(x))/289 + 240*x**(1/4)*cos(log(x))/289 + (16*x**(1/4)*sin(log(x))/17 + 4*x**(1/4)*cos(log(x))/17)*log(x)
     assert manualintegrate(f, x) == F and F.diff(x).equals(f)
+
+
+def test_issue15494():
+    from sympy.abc import t, s
+
+    Z0 = Function('Z0')
+    Z1 = Function('Z1')
+
+    k01, k10= symbols('k01 k10', real=True, positive=True)
```

Comment:
The symbols `Z0`, `Z1`, `k01`, `k10` are not needed for the test. They should be omitted.